variables.cmake

```diff
@@ -59,8 +59,7 @@ elseif (APPLE)
     set(DESKTOP_APP_USE_GLIBC_WRAPS OFF)
 else()
     set(LINUX 1)
-    execute_process(COMMAND uname -m OUTPUT_VARIABLE machine_uname)
-    if (NOT ${machine_uname} MATCHES "x86_64" AND NOT ${machine_uname} MATCHES "aarch64")
+	if (CMAKE_SIZEOF_VOID_P EQUAL 4)
```

Comment:
Wrong intendation, the cmake files do use 4 spaces and not tabs.

Reply:
Fixed, thanks.